pulse_actions/handlers/route_functions.py

```diff
@@ -0,0 +1,18 @@
+import pulse_actions.handlers.treeherder_buildbot as treeherder_buildbot
+import pulse_actions.handlers.treeherder_resultset as treeherder_resultset
+import logging
+
+logging.basicConfig(format='%(levelname)s:\t %(message)s')
+LOG = logging.getLogger()
+
+def route(data, message, dry_run):
+    if 'action' in data:
+        if data["action"] == "backfill":
+            treeherder_buildbot.on_buildbot_event(data, message, dry_run)
+        elif data["action"] in ["trigger_missing_jobs", "trigger_all_talos_jobs"]:
+            treeherder_resultset.on_resultset_action_event(data, message, dry_run)
+        else:
+            # We don't want to take any action on 'retrigger' or 'cancel' events
+            LOG.debug("We currently only support 'backfill', 'trigger_missing_jobs', 'trigger_all_talos_jobs' for multiple listeners.")
```

Comment:
Are you saying that "if 'resultset_id' in data" -> the message is from th-resultset actions and "if 'job_id' in data" the message is from th-job-actions? If so we can use that to just do:
```
if 'resultset_id' in data:
    treeherder_resultset.on_resultset_action_event(data, message, dry_run)
elif 'job_id' in data:
    treeherder_buildbot.on_buildbot_event(data, message, dry_run)
else:
   LOG.error(exchanges not supported by router)
```

That way treeherder_buildbot deals with the messages we don't want and if we ever try to use route for a new exchange without updating it first we get an error message.

The problem with this approach is if a completely unrelated exchange happens to have the same key and we somehow end up passing it to  route, but that sounds unlikely and we will get an error somewhere down the line.

But this is mostly to make it easier to attach other exchanges when they are needed, so if you want to keep like it is and change things later if it becomes necessary that is OK with me.